pyproject.toml

```diff
@@ -8,3 +8,6 @@ ignore = ["E203", "E731", "E501", "E741"]  # remove "E501", "E741" later
 [tool.ruff.format]
 # Enable reformatting of code snippets in docstrings.
 docstring-code-format = true
+
+[tool.bandit]
+exclude_dirs = ["improver_tests"]
```

Comment:
Why do we exclude this?

Reply:
I'll follow your lead.

Reply:
I think you were right to ask.  I have added an assert exclusion instead to the tests 👍